site2/docs/cookbooks-retention-expiry.md

```diff
@@ -31,7 +31,7 @@ By default, when a Pulsar message arrives at a broker it will be stored until it
 
 Retention policies are particularly useful if you intend to exclusively use the Reader interface. Because the Reader interface does not use acknowledgements, messages will never exist within backlogs. Most realistic Reader-only use cases require that retention be configured.
 
-When you set a size limit of, say, 10 gigabytes, then messages in all topics in the namespace, *even acknowledged messages*, will be retained until the size limit for the topic is reached; if you set a time limit of, say, 1 day, then messages for all topics in the namespace will be retained for 24 hours.
+When you set a size limit of, say, 10 gigabytes, then acknowledged messages in all topics in the namespace will be retained until the size limit for the topic is reached; if you set a time limit of, say, 1 day, then acknowledged messages for all topics in the namespace will be retained for 24 hours. The retention settings do not affect unacknowledged messages -- these are instead controlled by the backlog quota and retention policy.
```

Comment:
> The retention settings do not affect unacknowledged messages

While this is 100% correct in the context of a subscription, we should also clarify that retention will also keep messages (for the configured time/size) even if there's no subscription in the topic. 

Additionally, it will prevent the topic from being auto-deleted if not used until all the messages have expired the retention period.

Reply:
If I'm reading that correctly, that means there are 3 cases:

1) Messages not in a subscription backlog, and not acked: retention policy applies.
2) Messages in a subscription backlog, and not acked: backlog policy applies.
3) Messages acked by one or more subscriptions: retention policy applies.

Is that right? If so, then I think it might be worth just adding those three cases into the docs.

Reply:
Yes, all 3 points are 100% correct. I'm just thinking about the most unambiguous wording, since I understand it can be confusing..

> Messages not in a subscription backlog, and not acked 

A backlog exist only in the context of a subscription, so for a message to not be in a backlog and not be acked --> the only possible reason is that there are no subscriptions on the topic

> Messages in a subscription backlog, and not acked: backlog policy applies.

This I think is crystal clear

Reply:
Ok, I've taken a shot at clarifying the wording, please review the updates in a7ad1996d8.